### PR TITLE
Adds a button for CSV import option

### PR DESCRIPTION
**Pull request by seatedscribe**
## Description

In welcome window there is no button for the CSV import feature so I added it at the bottom of the pile.
It seems also that this feature has been missing from #292 last commit so I started from clear.
## How has this been tested?

Manual tests ( with ASAN of course :smiley: )

## Types of changes

- ✅ New feature (non-breaking change which adds functionality)

## Checklist:

- ✅ I have read the **CONTRIBUTING** document. **[REQUIRED]**
- ✅ My code follows the code style of this project. **[REQUIRED]**
- ✅ All new and existing tests passed. **[REQUIRED]**
- ✅ I have compiled and verified my code with `-DWITH_ASAN=ON`. **[REQUIRED]**

### CODE REVIEW

**Comment by TheZ3ro** [comment]
Can you use an horizontal layout for the 2 import button? Like this (removed) code https://github.com/keepassxreboot/keepassxc/pull/292/commits/5c80c31a702365b3f763ba451cc5dcaea3e3a706#diff-7c9786c8153007e5377a07f729774788L151 (from line 151 to 169 of WelcomeWidget.ui)

**Comment by seatedscribe** [comment]
Yeah, I got to give a look at your proposed layout at the time, and today as well. Both times I found the horizontal layout a bit confusing, and so decided to propose an alternative. 

What are the reasons of this flat layout being inappropriate? I'd like to share our impressions and thoughts.

My solution -one day will see the light, maybe- would be quite different, involving icons to get to the wanted function without the need to read all that text, but this is another story... :smile:

File: src/gui/WelcomeWidget.ui [hunk 1]
```diff
@@ -128,6 +128,13 @@
     </widget>
    </item>
    <item>
+    <widget class="QPushButton" name="buttonImportCSV">
+     <property name="text">
+      <string>Import from plain text CSV file</string>
```
Comment (louib) [inline src/gui/WelcomeWidget.ui]: Simply `Import from CSV` would be more consistent with the other import option `Import from KeePass 1`.

**Comment by phoerious** [comment]
I would prefer two buttons side by side as well. It reduces the overall visual clutter, because we only have three rows and not four. Also the two import options are very closely related and should therefore be grouped together.

**Pull request merged**